charts/microgateway/values.yaml

```diff
@@ -373,17 +373,24 @@ redis:
   # redis.enabled -- Deploy pre-configured [Redis](#redis).
   enabled: false
   fullnameOverride: redis
-  securityContext:
```

Comment:
Please review the following changes carefully. With the Bitnami Redis chart upgrade, the default Redis parameters need to be mapped to new parameters from the Redis Helm Chart. For more information see: https://github.com/bitnami/charts/tree/master/bitnami/redis#upgrading